addons/sale_stock/sale_stock.py

```diff
@@ -271,6 +271,20 @@ def _run_move_create(self):
             vals.update({'sequence': self.sale_line_id.sequence})
         return vals
 
+    @api.cr_uid_context
+    def create(self, cr, uid, vals, context=None):
```

Comment:
You are right.. @api.model should work. but at that time not sure, sale is not migration 
and this create method using @api.model generated error. so we used that . 
but i tested now @api.model work.. 
Thanks